tests/phpunit/api/v3/JobProcessMailingTest.php

```diff
@@ -102,15 +102,6 @@ public function testBasic(): void {
     $this->assertEquals('Running', $mailing['status']);
     $this->_mut->assertRecipients($this->getRecipients(1, 2));
     $this->_mut->clearMessages();
-    // Now test forwarding from there
-    $queue = MailingEventQueue::get(FALSE)->execute()->first();
-    $this->callAPISuccess('Mailing', 'event_forward', [
-      'event_queue_id' => $queue['id'],
-      'job_id' => $queue['job_id'],
-      'hash' => $queue['hash'],
-      'email' => 'a@b.com',
-      'params' => ['body_html' => 'Hi there'],
-    ]);
```

Comment:
I'm not sure if the bits below are related to `event_forward`  or some broader test